cpp/src/community/ecg.cu

```diff
@@ -42,25 +43,20 @@ binsearch_maxle(const IndexType *vec, const IndexType val, IndexType low, IndexT
 }
 
 template <typename IdxT, typename ValT>
-__global__ void match_check_kernel(IdxT size,
-                                   IdxT num_verts,
-                                   IdxT *offsets,
-                                   IdxT *indices,
-                                   IdxT *permutation,
-                                   IdxT *parts,
-                                   ValT *weights)
+__global__ void match_check_kernel(
+  IdxT size, IdxT num_verts, IdxT *offsets, IdxT *indices, IdxT *parts, ValT *weights)
```

Comment:
What are you using "_**IdxT**_" rather than "**_index_t_**" ?  It seems like we are not using capitalized types anywhere else

Reply:
This is an existing kernel that has existed for a while.  All I did was reformat.

I'd like to defer this until I finish implementing MNMG ECG when I will be reworking all of this code.  (@seunghwak suggested I delete the kernel entirely).